conans/test/unittests/util/hashed_path_test.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+
+from conans.test.utils.test_files import temp_folder
+from conans.test.utils.test_files import wait_until_removed
+
+from conans.util.windows import hashed_redirect
+
+
+class HashedPathTest(unittest.TestCase):
+    def setUp(self):
+        self.path = "package_name/version/user/channel/export"
+        self.folder = temp_folder()
+
+    def tearDown(self):
+        wait_until_removed(self.folder)
```

Comment:
No need to remove the folder. The conan test suite is not removing the temporary f, they are stored in tmp or temporary folders, and removed by the system. We need to align and decide first on the best policy, because removing can sometimes fail (thus, the ``wait_until_removed``, which, by the way, will block the test suite in CI if for some reason some OS hold a resource open, which sometimes happen in Windows, specially), and it might also make the suite much slower.

Reply:
Fair point, also without a tearDown it's consistent with the rest of the test suites.  I removed the `tearDown` method.